docs/content/time_dependent.md

```diff
@@ -43,3 +45,36 @@ print(out_state)
 ```
 
 Note that Qadence makes no assumption on units. The unit of passed duration value $\tau$ must be aligned with the units of other parameters in the time-dependent generator so that the integral of generator $\overset{\tau}{\underset{0}{\int}}\mathcal{\hat{H}}(t){\rm d}t$ is dimensionless.
+
+# Noisy time-dependent Hamiltonian evolution
+
+For performing noisy time-dependent Hamiltonian evolution, one needs to specify the `noise_operators` corresponding to the jump operators used within the time-dependent Schrodinger equation solver method `SolverType.DP5_ME`:
+
+```python exec="on" source="material-block" session="getting_started"
+from qadence import X, Y, HamEvo, TimeParameter, FeatureParameter, run
+from pyqtorch.utils import SolverType
+import torch
+
+# Simulation parameters
+ode_solver = SolverType.DP5_ME  # time-dependent Schrodinger equation solver method
+n_steps_hevo = 500  # integration time steps used by solver
+
+# Define block parameters
+t = TimeParameter("t")
+omega_param = FeatureParameter("omega")
+
+# Arbitrarily compose a time-dependent generator
+generator_td = omega_param * (t * X(0) + t**2 * Y(1))
+
+# Create parameterized HamEvo block
+noise_operators = [torch.eye(4, dtype=torch.complex128)]
+hamevo = HamEvo(generator_td, t, noise_operators = noise_operators)
```

Comment:
Hmm I think the interface should be to pass Qadence blocks, no?

E.g. `noise_operators = [I(0) @ I(1))]`. What are the operators users will want to pass here, various combinations of Paulis?

Reply:
Apparently, there are no restrictions on these noise matrices besides having the correct shape.

Reply:
I can of course allow for a list of blocks that would require a call `block_to_tensor` and removing the batchsize.

Reply:
My main question would be in practice what do users typically use. If it's mainly pauli strings, then it would be much more convenient to do it with Qadence blocks. If they want to use arbitrary matrices, it could also be a use for the `MatrixBlock`, which has not been used that much.

Reply:
Btw, it should allow projectors as well. But I guess you will make it work with any block right?

Reply:
@jpmoutinho Yes, btw I already modified it to work with a list of `AbstractBlock`.